raster-foundry/raster-foundry
# add landsat 4 5 7 product datasources

## Overview
Add products for landsat 4,5,7 as datasources

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~


## Testing Instructions
 * Run migration
* Verify that the new datasources show up
* Verify that the composites exist on each new datasource by viewing each datasource's details
* Verify that the bands and composites make sense using https://landsat.usgs.gov/what-are-band-designations-landsat-satellites.  Composites for landsat 4/5 were extrapolated from landsay 7/8

Closes #3345
